js/app.js

```diff
@@ -0,0 +1,44 @@
+ const a = 4;
+
+ const b = 53;
+ 
+  const c = 57;
+  
+  const d = 16;
+
+  const e = 'Kevin';
+
+  a < b;
+c > d;
+'Name' _=_ 'Name';
+a + b = c;
```

Comment:
same here—`=` is an assignment operator.  you can't assign a value to `a + b`

make sure you're trying these in your console! also, so, like: 

`console.log(a + b = c)` <--- paste that into the developer tools console in google chrome, make sure it does what you think it should do.  you should get an error.... how can you fix the error (try another operator)